packages/contracts/test/ProxyBorrowerWrappersScript.js

```diff
@@ -333,371 +324,6 @@ contract('BorrowerWrappers', async accounts => {
     th.assertIsApproximatelyEqual(ICRAfter, ICRBefore)
     // check that Stability Pool deposit
     th.assertIsApproximatelyEqual(depositAfter, depositBefore.sub(expectedLUSDLoss_A).add(netDebtChange))
-    // check lqty balance remains the same
-    th.assertIsApproximatelyEqual(lqtyBalanceAfter, lqtyBalanceBefore)
-
-    // LQTY staking
-    th.assertIsApproximatelyEqual(stakeAfter, stakeBefore.add(expectedLQTYGain_A))
-
-    // Expect Alice has withdrawn all ETH gain
-    const alice_pendingETHGain = await stabilityPool.getDepositorETHGain(alice)
-    assert.equal(alice_pendingETHGain, 0)
-  })
-
-
-  // --- claimStakingGainsAndRecycle ---
-
-  it('claimStakingGainsAndRecycle(): only owner can call it', async () => {
```

Comment:
keep test for now? seems like it's relevant to LQTY token removal patch instead

Reply:
this is removed as the LQTY events have been removed from stability pool state changes